File: template-springboot/src/main/java/com/example/demo/curd/entity/BaseEntity.java
package com.example.demo.curd.entity;

import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;
import java.time.LocalDateTime;

public class BaseEntity {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;

    private LocalDateTime createTime;
    private LocalDateTime updateTime;
    private String createUser;
    private String updateUser;
    private boolean deleted;

}

File: template-nestjs/src/entities/base.entity.ts
import { Expose } from "class-transformer";
import { Column, Entity } from "typeorm";


export class BaseEntity {
    

    @Column()
    create_time:Date;
    @Column()
    update_time:Date;
    @Column()
    create_user:string;
    @Column()
    update_user:string;
    @Column()
    deleted:boolean

}